source/clear-linux/guides/maintenance/kernel-development.rst

```diff
@@ -0,0 +1,335 @@
+to provide access to hardware and software functionality. 
+The Linux kernel is very customizable and extensible. 
+ 
+The |CL| kernels aim to be performant and practical. In some cases, it may be 
+necessary modify the kernel to suit your specific needs or test new kernel 
+code as a developer.
+
+In cases where drivers beyond those enabled by default in |CL-ATTR| are
+needed it may be necessary to:
+
+.. contents:: :local:
+   :depth: 2
+   :backlinks: top
+
+
+Request the change be included with the |CL| kernel
+===================================================
+
+If the kernel modification you need is already open source and likely to be 
+useful to others, consider submitting a request to add or enable in the
+|CL| kernel.
+
+If your change request is accepted, you do not need to maintain your own modified kernel.
+
+Make enhancement requests to the |CL| distribution `on GitHub`_ .
+
+
+Customize the Linux kernel source
+=================================
+
+In some cases, it may be necessary modify the kernel to suit your specific 
+needs or test new kernel code as a developer.
+
+You can build and install a custom kernel, however you must:
+
+* Disable secure boot
+* Maintain any updates to the kernel going forward
+
+To customize the kernel you will need a |CL| development evnironment, 
+make changes to the kernel, build the kernel, and install it.
+
+
+Install the |CL| development tooling framework
+----------------------------------------------
+
+.. include:: autospec.rst
+   :start-after: install-tooling-after-header:
+   :end-before: install-tooling-end:
+
+
+
+Clone the Linux kernel package 
+------------------------------
+clone the existing kernel package repository from |CL| as a starting point.
+
+#. Clone the Linux kernel package from |CL|.
+
+   .. code-block:: bash
+
+      cd ~/clearlinux
+      make clone_linux
+
+#. Navigate into the cloned package directory.
+
+   .. code-block:: bash
+
+      cd ~/clearlinux/packages/linux
+
+.. note::
+    The "linux" package is the kernel that comes with |CL| in the kernel-native bundle.
+    You can alternatively use a different kernel variant as the base for modification. 
+    For a list of kernel package names which you can clone instead, see the `clearlinux-pkgs GitHub`_.
+
+
+Change the kernel version 
+-------------------------
+
+|CL| tends to use the latest kernel available from kernel.org, the Linux 
+upstream. The kernel version that will be built can be changed in the 
+RPM SPEC file.
+
+.. note:: 
+   While most packages in Clear Linux are typically packaged using 
+   :ref:`autospec`, the kernel is not. This means control files provided 
+   by autospec are not available  and changes must be made directly.
+
+#. Open the Linux kernel package RPM SPEC file in an editor.
+
+   .. code-block:: bash
+
+      $EDITOR linux.spec
+
+#. Modify the Version, Release, and Source0 URL entries at the top of the file.
```

Comment:
What should I change these 3 fields to?  I would give an example of before and after.  Do I need to stay within a specific convention for version and release?  For source0, since I'm making a local change would I point to my local source?

Reply:
Including where to find valid kernel versions makes a lot of sense. 

I was hoping to using the `emphasize-lines` option in the [code-block directive](http://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#directive-code-block ) to make it more apparent which lines to change but couldn't get it quite working. 

I'm hoping @mvincerx and team can help with a solution after the tech review is complete.